arviz/tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+"""Configuration for test suite."""
+# pylint: disable=redefined-outer-name
+import os
+import pytest
+
+
+def pytest_addoption(parser):
+    """Definition for command line option to save figures from tests."""
+    parser.addoption("--save", nargs="?", const="test_images",
+                     help="Save images rendered by plot")
+
+
+@pytest.fixture(scope="session")
+def save_figs(request):
+    """Enable command line switch for saving generation figures upon testing."""
+    fig_dir = request.config.getoption("--save")
```

Comment:
it looks like if just `--save` is passed, `fig_dir` will be `None`, and nothing will get saved. Am I reading that right?

Reply:
It'll default to a directory named test_images later. I did it this way to save users from typing in a a folder name if they didn't care, but if they wanted to do a side by side comparion they could check out one branch and name the directory `before_cool_feature` and then checkout their other branch and name it `after_cool_feature` to have both plots

Hope that makes sense